### PR TITLE
[IMP] account: Add link to statement lines used to match a payment

### PR DESCRIPTION
**Pull request by smetl**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/demo/account_demo.py [hunk 1]
```diff
@@ -176,7 +175,7 @@ def _get_demo_data_reconcile_model(self):
                             self.env.company,
                         ).id,
                         'amount_type': 'regex',
-                        'amount_string': r'BRT: ([\d,]+)',
+                        'amount_string': f'BRT: ([\\d,.]+)',
```
Comment (FlorianGilbert) [inline addons/account/demo/account_demo.py]:
```suggestion
                        'amount_string': r'BRT: ([\\d,.]+)',
```

**Review by FlorianGilbert — APPROVED** [review]
@robodoo r+

**Comment by smetl** [comment]
@robodoo rebase-ff

**Comment by FlorianGilbert** [comment]
@robodoo rebase-ff

**Comment by FlorianGilbert** [comment]
@robodoo r+

**Comment by smetl** [comment]
@robodoo retry

**Pull request closed without merging**